GitUI/BranchTreePanel/RepoObjectsTree.Designer.cs

```diff
@@ -562,79 +577,96 @@ private void InitializeComponent()
+            this.mnubtnSynchronizeSubmodules.Size = new System.Drawing.Size(161, 26);
             this.mnubtnSynchronizeSubmodules.Text = "Synchronize";
             this.mnubtnSynchronizeSubmodules.ToolTipText = "Synchronize selected submodule recursively";
             // 
             // mnubtnResetSubmodule
             // 
             this.mnubtnResetSubmodule.Image = global::GitUI.Properties.Images.ResetWorkingDirChanges;
             this.mnubtnResetSubmodule.Name = "mnubtnResetSubmodule";
-            this.mnubtnResetSubmodule.Size = new System.Drawing.Size(138, 22);
+            this.mnubtnResetSubmodule.Size = new System.Drawing.Size(161, 26);
             this.mnubtnResetSubmodule.Text = "&Reset";
             this.mnubtnResetSubmodule.ToolTipText = "Reset selected submodule";
             // 
             // mnubtnStashSubmodule
             // 
             this.mnubtnStashSubmodule.Image = global::GitUI.Properties.Images.Stash;
             this.mnubtnStashSubmodule.Name = "mnubtnStashSubmodule";
-            this.mnubtnStashSubmodule.Size = new System.Drawing.Size(138, 22);
+            this.mnubtnStashSubmodule.Size = new System.Drawing.Size(161, 26);
             this.mnubtnStashSubmodule.Text = "&Stash";
             this.mnubtnStashSubmodule.ToolTipText = "Stash changes in selected submodule";
             // 
             // mnubtnCommitSubmodule
             // 
             this.mnubtnCommitSubmodule.Image = global::GitUI.Properties.Images.RepoStateDirtySubmodules;
             this.mnubtnCommitSubmodule.Name = "mnubtnCommitSubmodule";
-            this.mnubtnCommitSubmodule.Size = new System.Drawing.Size(138, 22);
+            this.mnubtnCommitSubmodule.Size = new System.Drawing.Size(161, 26);
             this.mnubtnCommitSubmodule.Text = "&Commit";
             this.mnubtnCommitSubmodule.ToolTipText = "Commit changes in selected submodule";
             // 
             // menuAllSubmodules
             // 
+            this.menuAllSubmodules.ImageScalingSize = new System.Drawing.Size(20, 20);
             this.menuAllSubmodules.Name = "contextmenuSubmodules";
             this.menuAllSubmodules.Size = new System.Drawing.Size(61, 4);
             this.menuAllSubmodules.Opening += new System.ComponentModel.CancelEventHandler(this.contextMenu_Opening);
             // 
+            // showCheckedLocalBranchesInRevisionGridButton
+            // 
+            this.showCheckedLocalBranchesInRevisionGridButton.Enabled = false;
+            this.showCheckedLocalBranchesInRevisionGridButton.Image = global::GitUI.Properties.Images.ShowThisBranchOnly;
+            this.showCheckedLocalBranchesInRevisionGridButton.Name = "showCheckedLocalBranchesInRevisionGridButton";
+            this.showCheckedLocalBranchesInRevisionGridButton.Size = new System.Drawing.Size(271, 26);
+            this.showCheckedLocalBranchesInRevisionGridButton.Text = "Show checked branches only";
```

Comment:
The same string twice -> please add it to Translated strings (could be a common TranslationString in this module only, but you could use this in menus later).

Reply:
Don't quite know what you want me to do here. I've used the existing "Show this branch only" menu item as a template, the text of which is also duplicated for the two different branch tree context menus. I didn't want to manually change the designer code either, as I expect those changes to be overwritten by the next designer changes. If you want me to implement it differently, please point me to a better example.